class.json-api-endpoints.php

```diff
@@ -1265,12 +1266,20 @@ function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
 			$response['exif']   = $metadata;
 		}
 
+		$is_video = false;
```

Comment:
I'll leave that up to you since you suggested it. Personally, I think this is a limited use case, I simply moved this out of a direct if/then to make it a variable that is checked for readability reasons.